include/zephyr/net/dhcpv6.h

```diff
@@ -38,11 +38,14 @@ enum net_dhcpv6_state {
 } __packed;
```

Comment:
Looks good, minor nit about the commit subject which could now be set to
`net: dhcpv6: Add configurable DUID buffer length`
no need to mention the config component there.